app/helpers/injection_helper.rb

```diff
@@ -4,7 +4,7 @@ module InjectionHelper
   include SerializerHelper
 
   def inject_enterprises(enterprises = nil)
-    inject_json_ams(
+    inject_json_list(
```

Comment:
Maybe `inject_json_array` might be a bit more idiomatic than `inject_json_list`?

Reply:
:+1: this is done in https://github.com/openfoodfoundation/openfoodnetwork/pull/5919